### DISCUSSION TITLE
export predefined resolvers

### DISCUSSION THREAD

**Discussion by Jisay** [hannoeru/vite-plugin-pages#246] [Ideas]
Hello!

Is it a reason why the vue, react and solid resolvers (https://github.com/hannoeru/vite-plugin-pages/tree/main/src/resolvers) are not exported by the module?
In my case I would like to use reactResolver in order to simply override stringify methods but I can't import it in my vite config. Only ts types can be imported.
It would be nice to export resolvers as well.

What do you think? Thanks!

**Comment by hannoeru** [comment]
Done, checkout https://github.com/hannoeru/vite-plugin-pages/releases/tag/v0.24.3

**Reply by Jisay** [reply]
Thanks a lot!
